samples/deviceCodeSample/src/main/java/deviceCodeFlowMain.java

```diff
@@ -0,0 +1,51 @@
+import com.azure.identity.DeviceCodeCredential;
+import com.azure.identity.DeviceCodeCredentialBuilder;
+import com.microsoft.graph.authentication.TokenCredentialAuthProvider;
+import com.microsoft.graph.exceptions.AuthenticationException;
+import com.microsoft.graph.httpcore.HttpClients;
+import okhttp3.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class deviceCodeFlowMain {
+
+    //Replace CLIENT_ID with your own client id from an adequately configured app
```

Comment:
what does "adequately configured app" mean? Is it an app with the requirements below? If so, maybe we can rephrase:
`from an app that is configured according to the requirements below`.

Reply:
```suggestion
    //Replace CLIENT_ID with your own client id from an app that is configured according to the requirements below
```